joomla/joomla-cms
# SEF for feeds

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/7437.
#### Summary of Changes

This PR adds the SEF plugin conversion to rss/atom feeds.
###### Before

![image](https://cloud.githubusercontent.com/assets/9630530/15226022/eda31aa4-1878-11e6-8bc9-a4d352742ef4.png)
###### After

![image](https://cloud.githubusercontent.com/assets/9630530/15225872/427534f0-1878-11e6-87da-270f26898f92.png)
#### Testing Instructions
1. Use latest staging
2. Create a Category List or blog menu item to list a category
3. Edit one of the articles of that category and add some internal links (+Article in editor) and images.
4. Ativate SEF system plugin and SEF in global config
5. Go to frontend to the category page (created at 1). Check all internal URL are converted to SEF URL
6. Add `?type=rss&format=feed` to the category URL to view the feed.
7. Notice the URL are not SEF in the feed.
8. Apply patch
9. Repeat step 5 and 6, notice the feed internal URL are now converted to SEF URL.
10. Do a general test to check if the SEF URL are working properly in the site and nothing is broken.
11. Test also with different combinations of SEF in global config and with/without multilanguage (language filter plugin).
